src/App/App.js

```diff
@@ -33,6 +33,7 @@ function App() {
     console.log("showMovieDetails CLICK!")
     // setSelectedMovie(movie);
     setSelectedMovie(movieDetails);
+    // ^ hardcoding Spirited Away movie details
```

Comment:
Once I pull down i will check the state of this quality, if need be I can try to work on generating each movies specific details